src/main/java/Duke.java

```diff
@@ -1,10 +1,212 @@
+import java.util.Scanner;
+
 public class Duke {
+    protected static Task[] tasks = new Task[100];
+    protected static int numTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        acceptCommand();
+        exit();
+    }
+
+    public static void greet() {
+        printLine();
+        System.out.println("Hi there! I'm Domo the chat bot.\nWhat would you like to do?");
+    }
+
+    public static void exit() {
+        printLine();
+        System.out.println("Goodbye. Hope to see you again soon!");
+        printDomo();
+    }
+
+    public static void acceptCommand() {
+        String input = getInput();
+        while (!isBye(input)) {
+            String command = getCommand(input);
+            if (isValidCommand(command)) {
+                executeCommand(input);
+            } else {
+                System.out.println("Not a valid command. Please try again.");
+            }
+            input = getInput();
+        }
+    }
```

Comment:
Since this method accepts more than one command (and, in fact, processes them), would a name like `acceptAndProcessCommands` be better?

Reply:
Will definitely think of a more fitting name for the method